src/Database/Redis/ManualCommands.hs

```diff
@@ -1159,6 +1193,9 @@ data XInfoStreamResponse
     { xinfoStreamLength :: Integer
     , xinfoStreamRadixTreeKeys :: Integer
     , xinfoStreamRadixTreeNodes :: Integer
+    , xinfoMaxDeletedEntryId :: Maybe ByteString -- since redis 7.0
```

Comment:
This is a comment let's make it a haddock:
```suggestion
    , xinfoMaxDeletedEntryId :: Maybe ByteString -- ^ Since redis 7.0
```

The same advice applies to all other occurrences.